### DISCUSSION TITLE
NPMJS Tokens compromised

### DISCUSSION THREAD

**Discussion by jasonkarns** [npm/feedback#901] [Registry]
After the LastPass breach, how would one go about marking registry auth tokens as compromised? I have the tokens but no way of knowing which accounts they are for.

How would one go about invalidating a token _given only the token_?

**Comment by ljharb** [comment]
Can you put it in your npmrc, and run `npm whoami`?

**Reply by jasonkarns** [reply]
was worth a shot:

```
$ npm whoami
npm ERR! code E401
npm ERR! 401 Unauthorized - GET https://registry.npmjs.org/-/whoami

npm ERR! A complete log of this run can be found in: /Users/jasonkarns/.npm/_logs/2023-04-08T15_55_34_778Z-debug-0.log
```

Still, would be good if there were a place to submit tokens to directly invalidate them (similar to how github's token invalidation works when they're detected in a repo).

**Reply by ljharb** [reply]
lol you could push them to a public repo and github will automatically invalidate them for you :-p

**Reply by jasonkarns** [reply]
Yeah I was half considering that. My paranoid half prevailed. 😆
